CoffeeChat/SceneDelegate.swift

```diff
@@ -21,14 +22,24 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
         let window = UIWindow(windowScene: scene)
         guard let signIn = GIDSignIn.sharedInstance() else {
-            window.rootViewController = LoginViewController()
+            let navigationController = UINavigationController(rootViewController: LoginViewController())
+            window.rootViewController = navigationController
             return
         }
         if signIn.hasPreviousSignIn() {
             signIn.restorePreviousSignIn()
-            window.rootViewController = OnboardingPageViewController(transitionStyle: UIPageViewController.TransitionStyle.scroll, navigationOrientation: UIPageViewController.NavigationOrientation.horizontal)
+            // Onboard user if they haven't done so yet, otherwise bring to home.
+//            let onboardingCompleted = userDefaults.bool(forKey: Constants.UserDefaults.onboardingCompletion)
+            let onboardingCompleted = false // Manually set onboardingCompleted to false to pull up onboarding.
```

Comment:
doesn't this make onboarding shown every time?

Reply:
`let onboardingCompleted = userDefaults.bool(forKey: Constants.UserDefaults.onboardingCompletion) `
is the code that will actually be used. 

I'm manually setting `onboardingCompleted` to false for now because I want to be able to see the onboarding screens. Otherwise, I will go straight to the home back every time.